web/login.html

```diff
@@ -58,9 +58,9 @@
 
 		// load main module
 		app.require('main', function () {
-			$('#login, #password').on('keypress', function () {
+			$('#login, #password').on('keypress', function (e) {
 				$('#message').html('&nbsp;'); 
-				if (event.keyCode == 13) $('#submit').click();
+				if (((typeof e.which == "number") ? e.which : e.keyCode) == 13) $('#submit').click();
```

Comment:
You are right for keyCode == 13 works for all browsers.
But for Mozilla browser on characters/numbers  don't work correctly (tested in Win8)  - keyCode == 0.  Can be tested at  http://javascript.info/tutorial/keyboard-events